src/ucp/core/ucp_ep.c

```diff
@@ -219,26 +228,27 @@ ucs_status_t ucp_ep_create(ucp_worker_h worker,
         goto err_destroy_ep;
     }
 
+    if (err_mode == UCP_ERR_HANDLING_MODE_PEER) {
+        /* Disable RNDV */
+        ucp_ep_config(ep)->rndv.am_thresh = SIZE_MAX;
```

Comment:
this should be done inside ucp_ep_config_init() - outside this function, the config should not be modified.